may13th.ts

```
@@ -0,0 +1,103 @@
+}
+
+interface FixesRequestParams {
+  key: string
+}
+
+interface VerdictRequestBody {
+  key: string
+  selectedFix: number
+}
+
+export const serveCodeFixes = () => (req: Request<FixesRequestParams, {}, {}>, res: Response, next: NextFunction) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)